packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -346,8 +357,8 @@ const expressMinimalPrompts = () => {
     return inquirer.prompt(questions)
 }
 
-const generateExpressMinimal = (projectId, {outputDir, verbose}) => {
-    extractTemplate('template-express-minimal', outputDir)
+const generateTemplateMinimal = (projectId, {outputDir, verbose}, template) => {
```

Comment:
Nit Question: What is the reasoning for renaming, looking forward I'd expect anything that the generated creates is from a template, so it might be wasted characters.

Reply:
The reason for the renaming is to make the `generateExpressMinimal` method generic for all minimal templates, thus `generateTemplateMinimal`.  But I'm open to any other name. Does `generateTemplate` sound better?

Reply:
Ok.. so looking at the code, `generateTemplateMinimal` is essentially a slimmed down version of `runGenerator` right? It looks like we are those 2 code paths are diverging, I'm not sure if this is ideal but you are following the current pattern so its always something we can come back to fix. 

Just one note. `generateTemplateMinimal` can technically be used for any template that doesn't have complex question/answer substitutions. But you are using "minimal" like it will only work for those templates called minimal which isn't necessary. I'd try to make the name similar to `runGenerator` but imply that it's doing the least work. 

But I'm starting to ramble, we can talk in meet if you have concerns.